aws-android-sdk-cognitoidentityprovider-test/src/androidTest/java/com/amazonaws/mobileconnectors/cognitoidentityprovider/CognitoUserIntegrationTest.java

```diff
@@ -37,6 +38,7 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
+@Ignore
```

Comment:
Can we slap an `@Ignore("Justification clause...")` on these puppies?

Reply:
Can I swear?

I kid! Yes! Will do.